.storybook/main.ts

```diff
@@ -0,0 +1,88 @@
+const path = require("path");
+const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
+import type { StorybookConfig } from "@storybook/react-webpack5";
+const config: StorybookConfig = {
+  stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
+  core: {},
+  addons: ["@storybook/addon-links", "@storybook/addon-essentials", "@storybook/preset-create-react-app", "@storybook/addon-interactions"],
+  framework: {
+    name: "@storybook/react-webpack5",
+    options: {
+      builder: {
+        lazyCompilation: true
+      }
+    }
+  },
+  docs: {
+    autodocs: "tag"
+  },
+  staticDirs: ["../build"],
+  webpackFinal: async config => {
+    if (config.resolve && config.resolve.alias) {
+      const {
+        global,
+        ...alias
+      } = config.resolve.alias;
+      // config.resolve.alias['global'] = undefined;
+      // const { ...alias } = config.resolve.alias
```

Comment:
Suggest removing if not needed or add another comment/note about what they are and why commented out